README.md

```diff
@@ -246,8 +246,9 @@ Mac app wrapper around Google's stand-alone Android Messenger. ![javascript_icon
 - [Now Desktop](https://github.com/zeit/now-desktop) - Create deployments right from the tray menu. ![javascript_icon]
 - [stts](https://github.com/inket/stts) - macOS app for monitoring the status of cloud services. ![swift_icon]
 
-### Other
+#### Other
```

Comment:
Why do you add a '#' symbol? Please remove this change.

Reply:
In your TOC this section is a subheading, so looks like it’s misformatted here. Happy to revert if that’s not the case.

Reply:
Updated 👍

Reply:
> In your TOC this section is a subheading, so looks like it’s misformatted here. Happy to revert if that’s not the case.

I see. That's a good point. I'll fix that. Thank that you mention that!